jupyterlab/labbox_ephys_widgets_jp/webpack.config.js

```diff
@@ -38,6 +38,9 @@ module.exports = [
     devtool: 'source-map',
     externals,
     resolve,
+    node: {
+      fs: 'empty'
```

Comment:
This was needed because of trouble with raw.macro... but I now no longer use raw.macro anyway -- instead uses md.gen.ts system as described above.